src/ConnectedMode/Migration/ConnectedModeMigration.cs

```diff
@@ -194,9 +185,38 @@ private System.Threading.Tasks.Task<string> GetFileContentAsync(string filePath)
             return changedFiles;
         }
 
-        private async Task SaveChangedFilesAsync(ChangedFiles changedFiles)
+        private async Task MakeLegacyFileChangesAsync(LegacySettings legacySettings, ChangedFiles changedFiles,
```

Comment:
Logically the code is the same as before i.e. I haven't changed what it does, just the order it does it in.